README.md

```diff
@@ -34,7 +34,9 @@ and [contributing guidelines](CONTRIBUTING.md).
 ## Quickstart
 
 These steps describe a quick local setup.
-They are not suitable for most production use-cases because they keep all data inside containers.
+They are not suitable for most production use-cases because they keep all data
+inside containers and don't [encrypt incoming connections](https://docs.ferretdb.io/security/#securing-connections-with-tls/).
```

Comment:
Please use `../security.md`. Never include a domain name

Reply:
That's the README file, accessible from github view

Reply:
Oops! My bad, sorry